packages/live-preview-sdk/src/helpers/utils.ts

```diff
@@ -13,14 +13,14 @@ export function sendMessageToEditor(
   data: EditorMessage,
   targetOrigin: string[]
 ): void {
-  const message: MessageFromSDK = {
+  const message = {
     ...data,
     method,
     from: 'live-preview',
     source: LIVE_PREVIEW_SDK_SOURCE,
     location: window.location.href,
     version,
-  };
+  } as MessageFromSDK;
```

Comment:
out of curiosity, why?

Reply:
Some overlap/confusion inside of typescript due all the different message types.
I will create a story to improve the types for the messages in general as it could produce errors

Reply:
Okay cool, thanks for explaining!